static/app/components/profiling/flamegraphZoomView.tsx

```diff
@@ -782,4 +808,51 @@ const Canvas = styled('canvas')`
   position: absolute;
 `;
 
+// loosely based spreadsheet navigation
+const keyDirectionMap: Record<string, Direction> = {
+  ArrowUp: 'up',
+  ArrowDown: 'down',
+  ArrowLeft: 'left',
+  ArrowRight: 'right',
+  Tab: 'right',
+  'Shift+Tab': 'left',
+  Enter: 'down',
+  'Shift+Enter': 'up',
+  w: 'up',
+  s: 'down',
+  a: 'left',
+  d: 'right',
+};
+
+function handleFlamegraphKeyboardNavigation(
```

Comment:
Nit, but I wouldn't even dispatch to this fn if the key that is pressed is not a valid direction key, this could allow you to improve the signature a little bit by also checking if currentFrame exists.